static/app/components/group/suggestedOwners/ownershipRules.tsx

```diff
@@ -78,3 +144,51 @@ const HelpfulBody = styled('div')`
   padding: ${space(1)};
   text-align: center;
 `;
+
+const Container = styled(Panel)`
+  width: 100%;
+  height: 100%;
+  background: none;
+  display: flex;
+  flex-direction: column;
+  padding: ${space(2)};
+`;
+
+const HeaderContainer = styled('div')`
+  display: flex;
+  align-items: flex-start;
+`;
+
+const Header = styled('h4')`
+  margin-bottom: ${space(1)};
+  text-transform: uppercase;
+  font-weight: bold;
+  color: ${p => p.theme.gray400};
+  font-size: ${p => p.theme.fontSizeMedium};
+`;
+
+const Content = styled('span')`
+  color: ${p => p.theme.gray400};
+`;
+
+const Actions = styled('div')`
+  display: grid;
+  grid-template-columns: max-content auto;
+  justify-items: start;
+  align-items: end;
+  margin-top: ${space(2)};
+  grid-column-gap: ${space(1)};
+`;
+
+const DocsButton = styled(Button)``;
+
+const SetupButton = styled(Button)`
```

Comment:
Why this override?

Reply:
@EvanPurkhiser eh missed this in cleanup